src/Util.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Util where
+
+import           Prelude hiding (FilePath)
+import           Filesystem.Path
+import           Filesystem.Path.CurrentOS () -- https://github.com/fpco/haskell-filesystem/issues/11
+
+isBoring :: FilePath -> Bool
+isBoring p = ".git/" `elem` dirs || "dist/" `elem` dirs
```

Comment:
This could get back if e.g. there are undo, swap or backup files from editors in the current dir. It's more work, but maybe only reloading on changes to files listed in exposed modules, extra-modules, data-files, etc. in the cabal file would be better?